hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -32,6 +32,13 @@
             $scope.RecordsToDisplay = "10";
             $scope.currentPage = 1;
             $scope.lastIndex = 0;
+            $scope.statistical = {
+                usages : {}
+            }
+            $scope.statistical.usages.min = "";
+            $scope.statistical.usages.max = "";
+            $scope.statistical.usages.medina = "";
+            $scope.statistical.usages.stdev = "";
```

Comment:
Nit: `statistical` -> `statistics`.

I think it's also better to be more specific that it is the statistics for DNs (since technically we can have statistics for SCM in the future). Maybe `scope.statistics.nodes.usages.` prefix is better.

Reply:
I have updated it.